### ISSUE TITLE
Improve miniature for technical blog post

### ISSUE THREAD

**Issue by ferdi05** [meilisearch/devrel#383]
It's more understandable by our users, and should drive new users.
e.g.:
![Capture d’écran 2022-04-01 à 12 54 23](https://user-images.githubusercontent.com/2602288/161259741-223f173b-f917-4b08-9b1d-53fe2c0f5b0d.png)

**Comment by ferdi05** [comment]
Also a good image is very useful for the social media performance of our posts.
See some tips on how to write the best performing social media messages [on Notion](https://www.notion.so/meilisearch/Social-Media-Queue-f5ab593e972d49a891059e2e537be7fe#ad47b7f117ee4b44ba73e09a963c0139) _(private link)_.

**Comment by ferdi05** [comment]
We have a process on blog post cover creation [on Figma](https://www.figma.com/file/9Xe1cT0Feh8BQKaFO3LZ3e/Blog-cover?node-id=5%3A45).
We need to update our previous blog post cover when needed.

Content that require an update:

- [ ] Every [release blog posts](https://blog.meilisearch.com/tag/release/)
- [ ] Integrations blog posts: [Gatsby](https://blog.meilisearch.com/add-a-search-bar-to-your-gatsby-project-using-meilisearch/), [Strapi v3](https://blog.meilisearch.com/add-meilisearch-to-your-strapi-app/)
- [ ] Other tutorials: [React+Rails](https://blog.meilisearch.com/how-to-integrate-an-extremely-fast-and-relevant-search-into-your-rails-app-using-meilisearch-and-react/), [React](https://blog.meilisearch.com/how-to-implement-instant-search-within-5-minutes-in-your-react-app/)

**Comment by ferdi05** [comment]
Why a good blog post cover, quoted from [a RenderForm blog post](5 Tips for Creating Great Blog Post Covers):

> A blog post cover can make or break your content. It's the first thing readers see when they come across your post and it should be enticing enough for them to click, read, and share.

This could apply to blog posts that don't really have a cover, even if I'm not convinced this would fit with other illustrations, but I don't really have any better idea. We could also wait until we have a new blog template.
> While there are no general rules for what makes a great blog post cover, there are some pointers to keep in mind. Make sure you incorporate images if possible. Include the title in the image so that viewers know what they're clicking on. Use an image that is visually appealing and relevant to your topic the more eye-catching it is the more likely people will click on it. Make sure your images are high quality so that nothing looks pixelated or blurry. And lastly, don't forget to include a catchy headline with an enticement!

**Comment by ferdi05** [comment]
I just updated [some of the latest blog post covers](https://blog.meilisearch.com/) for new version releases, thanks to @lpinot great work [on Figma](https://www.figma.com/file/9Xe1cT0Feh8BQKaFO3LZ3e/Blog-cover?node-id=5%3A45) _(private link)_.
I like it, but I feel like if we change all the covers, this may look super uniform, like [Strapi's blog](https://strapi.io/blog) (I don't want to call names here, they're our friends and I hope they will forgive me for this).
wdyt @lpinot @CaroFG ?

**Comment by lpinot** [comment]
I don't think it is going to look too uniform. We obviously need something more consistent between the covers, but it doesn't mean it's going to look boring. We can bring some rhythm with a pattern depending on what type of article it is. 
E.g., all the release posts are in purple; tutorials are in blue, demos in dark blue, etc... + with sometimes a few illustrations, it's not possible to be boring. WDYT?

**Comment by CaroFG** [comment]
I totally agree with @lpinot. Talking about consistency, why is the background color different in the release posts? Also, I'd keep the version in bold 😄 but it's my aesthetic-blind-person opinion

**Comment by lpinot** [comment]
I agree @CaroFG regarding the different colors of the release posts, we should choose one and use the same each time

**Comment by ferdi05** [comment]
Thanks for your input @lpinot @CaroFG 
I agree that using illustrations sometimes could be really helpful to make it look less dull.

> I totally agree with @lpinot. Talking about consistency, why is the background color different in the release posts? Also, I'd keep the version in bold 😄 but it's my aesthetic-blind-person opinion

I was using some different proposition from @lpinot to indeed make it more colorful, but I'll be happy to apply the one color per content type strategy!
I'm changing it!

**Comment by CaroFG** [comment]
I am trying to create a list of different blog post types and establish which ones need a cover designed by @lpinot.

I have identified 8 types of blog posts:

- Release
- Tutorials / integrations
- Demos
- Life at Meilisearch
- Company
- Cloud
- Special events: hacktoberfest
- Other (one-off posts that do not fall into any of the preceding categories: posts about knowledge management, CFPs...)

**Comment by CaroFG** [comment]
For now, we have templates for the following categories:
- Release (purple dark background with stars and a 2-line title: the first line reads "What's new?", the second line contains the version number)
- Tutorials / integrations: logo of the language / framework... followed by a plus sign and the Meilisearch logo

Since we decided to have consistent in-brand covers, only one tutorial dedicated to a demo has been published: [Take control of Meilisearch's typo tolerance](https://blog.meilisearch.com/typo-tolerance/). The cover was designed by @lpinot. The image used includes 
- the title of the blog post : white and pink (pink used to highlight the name of the showcased feature)
-  an illustration representing the settings, which evoke the title (taking control). 

Next demos blog posts could follow the same pattern.

**Comment by CaroFG** [comment]
The 3 categories mentioned above —release, tutorials/integrations, and demos— are the most recurrent posts. We should decide on a design pattern to bring a little variety and avoid having a monochromatic blog while keeping consistency. I like the idea of using different background colors according to the post type. For now, the only color used has been Valhalla 500. I think we should keep this one for the release blog posts and use others for the demos and the tutorials.

**Comment by ferdi05** [comment]
I agree with all said here, thanks @CaroFG for your work here! Watching the progress.

**Comment by CaroFG** [comment]
Little reminder to @lpinot 😄

**Comment by ferdi05** [comment]
@CaroFG We made a lot of progress regarding post covers. Do you feel like there are still covers we should work on?

**Comment by CaroFG** [comment]
Yes, find the list below, I asked the design team on Slack to work on, them: 
- [How to Search Nobel Prize Winners Faster With MeiliSearch and JavaScript](https://blog.meilisearch.com/how-to-search-nobel-prize-winners-faster-with-meilisearch-api-for-javascript/)
-[ How to Speed Up GoodReads Book Titles Search With MeiliSearch and JavaScript](https://blog.meilisearch.com/how-to-speed-up-goodreads-book-titles-search-with-meilisearch-and-javascript/)
- [Save and Search Through Your Slack History on a Free Slack Plan](https://blog.meilisearch.com/slack-history/)
- [Add a search bar to your Gatsby project using Meilisearch](https://blog.meilisearch.com/add-a-search-bar-to-your-gatsby-project-using-meilisearch/)
-[ How to use Meilisearch in your multi-tenant application](https://blog.meilisearch.com/multi-tenancy/)

**Comment by CaroFG** [comment]
If we decide to create different covers for different type of content, we should look at the [tags](https://github.com/meilisearch/devrel/issues/466)
